changelog/db.changelog-1.2.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog" xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext" xmlns:pro="http://www.liquibase.org/xml/ns/pro" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+ <changeSet author="Simon Dunn" id="1">
+    <tagDatabase tag="v1.1.0" />
+  </changeSet>
+  <changeSet author="Simon Dunn" id="2">
+    <dropForeignKeyConstraint baseColumnNames="paymentRequestId" baseTableName="settlements" constraintName="fk_paymentRequests_paymentRequestId" referencedColumnNames="paymentRequestId" referencedTableName="paymentRequests" />
+  </changeSet>
+  <changeSet author="Simon Dunn" id="3">
+    <dropNotNullConstraint tableName="settlements" columnName="paymentRequestId" columnDataType="INTEGER"/>
+  </changeSet>
+  <changeSet author="Simon Dunn" id="4">
+    <addColumn tableName="settlements">
+      <column name="value" afterColumn="sourceSystem" type="INTEGER" />
+    </addColumn>
+  </changeSet>
```

Comment:
should `settlement` and `paymentRequest` be linked by `invoiceNumber` rather than prev `paymentRequestId` (which is internal microservice gumpins)?

How do we know which `paymentRequest` this particular `settlement` entry relates to?

Reply:
Can match to a paymentRequest using the `invoiceNumber` and the `value`.
Not sure if it needs to be a foreign key, what is the benefit of this?